src/pages/ManageUser/transactionHistory.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import { ReactComponent as Credit} from "../../assets/icons/credit.svg";
+import { ReactComponent as Debit} from "../../assets/icons/debit.svg";
+
+
+interface ITransact {
+    type: string;
+    text: string
+    details: string
+    amount: string
+    date: string
+}
+
+const transactions = [
+    {
+        type: "credit",
+        text: "Bank Transfer",
+        details: "770 TF ALESSA AB...",
+        amount: "₦9,430.15",
+        date: "14:37 | 10 AUG, 2023"
+    },
+    {
+        type: "debit",
+        text: "Bank Withdrawal",
+        details: "770 TF ALESSA AB...",
+        amount: "₦9,430.15",
+        date: "14:37 | 10 AUG, 2023"
+    },
+    {
+        type: "credit",
+        text: "Bank Transfer",
+        details: "770 TF ALESSA AB...",
+        amount: "₦9,430.15",
+        date: "14:37 | 10 AUG, 2023"
+    },
+    {
+        type: "debit",
+        text: "Bank Transfer",
+        details: "770 TF ALESSA AB...",
+        amount: "₦9,430.15",
+        date: "14:37 | 10 AUG, 2023"
+    },
+    
+]
+
+const EachTransaction = ({type, text, details, amount, date}:ITransact ) => {
+    return(
+        <div className='py-4 px-4 flex items-center justify-between border-b last:border-none border-[#E3E3E3]'>
```

Comment:
Move this to a sperate file and consider renaming it to Transaction rather than EachTransaction